distribution/src/conf/openfire-demoboot.xml

```diff
@@ -50,5 +50,8 @@
               </roster>
           </user2>
       </users>
+      <authprovider>
+          <mode>custom</mode>
+      </authprovider>
```

Comment:
Will this have an effect on other things that use demoboot?

Reply:
I don't think so, but it's a nasty hack. The best approach would be fixing OF-2592.

Reply:
I've now created https://github.com/igniterealtime/Openfire/pull/2222 to address OF-2592. If that gets merged before this PR, we could drop this hack.

Reply:
Merged. Reverted the hack & rebased.